embed-pdf-viewer.php

```diff
@@ -67,8 +67,6 @@ public function embed_pdf_media_editor( $html, $id ) {
 		if ( 'application/pdf' !== $post->post_mime_type ) {
 			return $html;
 		}
-
-
```

Comment:
That's what my ruleset in PhpStorm does and I don't see anything about it in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards